packages/@simple-dom/serializer/src/index.ts

```diff
@@ -15,18 +15,41 @@ const ESC: {
   '>': '&gt;',
 };
 
+const VALID_UPPER_TAGS: {
+  [upper: string]: string;
+} = {
+  DIV: 'div',
+  SPAN: 'span',
+};
+
+export enum WHITESPACE_TRIM_VALS {
+  LEFT = 'trimLeft',
+  RIGHT = 'trimRight',
+  ALL = 'trim',
+}
+
 function matcher(char: string) {
   return ESC[char];
 }
 
 function toLowerCase(name: string) {
-  return name === 'DIV' ? 'div' : name === 'SPAN' ? 'span' : name.toLowerCase();
```

Comment:
ya, this conditional seems odd to me also, even if `name` were `DIV` calling `name.toLowerCase()` would result in the value we already were returning 🤔 

@krisselden - Do you happen to recall why we did it this way?

Reply:
yes because of a benchmark, toLowerCase was a good chunk of the serialization cost and this avoided the most common cases